README.md

```diff
@@ -116,7 +116,7 @@ Here is an [example](Test_Data/INHS_FISH_000742.json) of a metadata file.
   
 ### Measure
 
-The output of the measure tool is a file named basename_measure.json that has measurements (in pixels) for each trait.
+The output of the measure tool is a file named fishARKID_measure.json that has measurements (in pixels) for each trait.
```

Comment:
Same thing here. Whatever the user passes along the command line will be the output filename.
```suggestion
The output of the measure tool is a JSON file that has measurements (in pixels) for each trait.
```